ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/GhcrIntegrationForm.tsx

```diff
@@ -0,0 +1,326 @@
+                                isDisabled={!isEditable}
+                            />
+                        </FormLabelGroup>
+                    )}
+                    <FormLabelGroup
+                        label="Password"
+                        labelIcon={
+                            <IntegrationHelpIcon
+                                helpTitle="GitHub password"
+                                helpText={
+                                    <Flex direction={{ default: 'column' }}>
+                                        <FlexItem>
+                                            A GitHub personal access token with at least{' '}
+                                            <em>read:packages</em> scope. For more information, see{' '}
+                                            <ExternalLink>
+                                                <a
+                                                    href="https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#authenticating-to-the-container-registry"
+                                                    target="_blank"
+                                                    rel="noopener noreferrer"
+                                                >
+                                                    Authenticating to the Container registry
+                                                </a>
+                                            </ExternalLink>
+                                        </FlexItem>
+                                    </Flex>
+                                }
+                                ariaLabel="Help for password"
+                            />
+                        }
+                        fieldId="config.docker.password"
+                        touched={touched}
+                        errors={errors}
+                    >
+                        <TextInput
+                            type="password"
+                            id="config.docker.password"
+                            value={values.config.docker.password}
+                            onChange={(event, value) => onChange(value, event)}
+                            onBlur={handleBlur}
+                            isDisabled={!isEditable || !values.updatePassword}
+                            placeholder={
+                                values.updatePassword
+                                    ? ''
+                                    : 'Currently-stored password will be used.'
+                            }
+                        />
+                    </FormLabelGroup>
+                    <FormLabelGroup
+                        fieldId="config.skipTestIntegration"
+                        touched={touched}
+                        errors={errors}
+                    >
+                        <Checkbox
+                            label="Create Integration Without Testing"
```

Comment:
For your awareness, not a change request, consistency wins here, but for new interface elements, **Sentence case** instead of **Title Case** according to PatternFly and Red Hat guideline:
> Default to sentence case across all UI elements, including navigation items, page titles, buttons, and so on.
https://www.patternfly.org/ux-writing/capitalization

Reply:
Ah I copied this from the ACR integration, but it seems most integrations today already use sentence case. I'll do a follow up PR to switch all existing integrations to sentence case.

Reply:
https://github.com/stackrox/stackrox/pull/13433